2017/SimBourse/src/core/Marche.java

```diff
@@ -391,6 +398,19 @@ public String toString() {
 	}
 
 	public void destroy(){
+		try {
+			Charset charset = Charset.forName("UTF-8");
+			Writer writer = new OutputStreamWriter(new FileOutputStream(new File("OperationsDernierePartie.txt")), charset);
+			for (Operation operation : liste_Operations){
```

Comment:
A nouveau pas besoin de for : String.valueOf(liste_Operations) suffit.
Le saut de ligne servira à différencier chaque partie.